src/js/cityDetails.js

```diff
@@ -0,0 +1,39 @@
+/* global document */
+
+const setUpDetails = (markerGroup, data) => {
+  const cityDetailsElement = document.querySelector(".city-details-popup");
+
+  // Clicking any city marker, updates HTML and makes popup visible
+  markerGroup.on("click", function (e) {
+    const cityState = e.sourceTarget.getTooltip().getContent();
+
+    cityInfo = createCityDetailsText(data, cityState);
+
+    document.querySelector(".city-details-text").innerHTML = cityInfo;
+
+    cityDetailsElement.style.display = "block";
```

Comment:
I don't think this whitespace is very useful. A blank line can be super valuable, but usually to separate unrelated blocks of code. Like these 3 lines are about X and the next 2 about Y. These all seem related